frontend/server/minio-helper.ts

```diff
@@ -126,13 +128,18 @@ function extractFirstTarRecordAsStream() {
  * @param param.bucket Bucket name to retrieve the object from.
  * @param param.key Key of the object to retrieve.
  * @param param.client Minio client.
+ * @param param.tryExtract Whether we try to extract *.tar.gz, default to true.
  *
  */
 export async function getObjectStream({
   bucket,
   key,
   client,
-}: MinioRequestConfig): Promise<Transform> {
+  tryExtract,
+}: MinioRequestConfig & { extract?: boolean }): Promise<Transform> {
```

Comment:
`{ extract?: boolean }` what is this line suppose to do? I don't see `extract` used anywhere? i.e. u already modified `MinioRequestConfig` with `tryExtract`